### PR TITLE
LOG-569: Static index config

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:
* moves viaq plugin into this repo
* Adds static index config
* fixes REPO_MAVEN_URL to use https

https://jira.coreos.com/browse/LOG-569

### CODE REVIEW

**Comment by richm** [comment]
lgtm - please squash and I'll `/lgtm`

**Comment by jcantrill** [comment]
block on https://github.com/openshift/origin-aggregated-logging/pull/1798

**Comment by richm** [comment]
@jcantrill do we still need the hold on this PR?

**Comment by richm** [comment]
hmm - now what's wrong with ci :-(
https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_origin-aggregated-logging/1797/pull-ci-openshift-origin-aggregated-logging-master-full-integ-aws/1642/artifacts/full-integ-aws/

fluentd is not starting - log: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_origin-aggregated-logging/1797/pull-ci-openshift-origin-aggregated-logging-master-full-integ-aws/1642/artifacts/full-integ-aws/fluentd-b9xlw.log
```
error: unable to upgrade connection: container not found ("fluentd")
```
describe: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_origin-aggregated-logging/1797/pull-ci-openshift-origin-aggregated-logging-master-full-integ-aws/1642/artifacts/full-integ-aws/fluentd-b9xlw.describe
```
  Normal   Started    11m (x5 over 13m)    kubelet, ip-10-0-129-4.ec2.internal  Started container fluentd
  Normal   Pulled     11m (x4 over 13m)    kubelet, ip-10-0-129-4.ec2.internal  Container image "registry.svc.ci.openshift.org/ci-op-mj1qpcyr/stable:logging-fluentd" already present on machine
  Warning  BackOff    4m9s (x43 over 13m)  kubelet, ip-10-0-129-4.ec2.internal  Back-off restarting failed container
```
@JAORMX what's your method for pulling the fluentd logs from the node directly?

**Comment by richm** [comment]
fluentd didn't start - might not be a flake
/retest

**Comment by richm** [comment]
@jcantrill this PR isn't going to pass until you can run origin-aggregated-logging CI on the feature-es6x branch with EO and CLO images which are also built from the feature-es6x branch

**Comment by jcantrill** [comment]
Broken on ` config error file="/etc/fluent/fluent.conf" error_class=Fluent::ConfigError error="Unknown filter plugin 'viaq_data_model'. Run 'gem search -rd fluent-plugin' to find plugins"`

File: fluentd/Dockerfile [hunk 1]
```diff
@@ -83,6 +83,7 @@ ADD lib/generate_throttle_configs/lib/*.rb ${HOME}/
 ADD lib/filter_parse_json_field/lib/*.rb /etc/fluent/plugin/
 ADD lib/filter_elasticsearch_genid_ext/lib/filter_elasticsearch_genid_ext.rb /etc/fluent/plugin/
 ADD lib/parser_viaq_host_audit/lib/*.rb /etc/fluent/plugin/
+ADD lib/fluent-plugin-viaq_data_model/lib/*.rb /etc/fluent/plugin/
```
Comment (richm) [inline fluentd/Dockerfile]: `ADD lib/fluent-plugin-viaq_data_model/lib/fluent/plugin/*.rb /etc/fluent/plugin/`

File: fluentd/Dockerfile.centos7 [hunk 2]
```diff
@@ -46,6 +46,7 @@ ADD lib/generate_throttle_configs/lib/*.rb ${HOME}/
 ADD lib/filter_parse_json_field/lib/*.rb /etc/fluent/plugin/
 ADD lib/filter_elasticsearch_genid_ext/lib/filter_elasticsearch_genid_ext.rb /etc/fluent/plugin/
 ADD lib/parser_viaq_host_audit/lib/*.rb /etc/fluent/plugin/
+ADD lib/fluent-plugin-viaq_data_model/lib/*.rb /etc/fluent/plugin/
```
Comment (richm) [inline fluentd/Dockerfile.centos7]: `ADD lib/fluent-plugin-viaq_data_model/lib/fluent/plugin/*.rb /etc/fluent/plugin/`

**Comment by ewolinetz** [comment]
looks good, squash and we can move it along

**Comment by jcantrill** [comment]
> looks good, squash and we can move it along

Purposely did not squash to make sure there are two obvious commits: one for bringing in the gem and the other for the change made to it.

**Pull request merged**